index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prangon's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logo-placeholder">
+                <div class="my-logo">
+                    PG
+                </div>
```

Comment:
- I think it will be better if you replace the `<div class="my-logo">` with an `<a>` tag because of three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info 🏵️ 

I think you can re-factor it to look like this instead 👇🏽 
```
<a href="#" class="my-logo" >PG</a>
```
How do you see this link of code. Often time, websites loge are meant to be a clickable link 👍🏽